### PR TITLE
Listing all possible options from phinx as valid option in the shell.

### PR DESCRIPTION
**Pull request by lorenzo**
Also locking version to something we know is stable for us.

### CODE REVIEW

**Comment by lorenzo** [comment]
Fixes https://github.com/cakephp/migrations/issues/51

**Comment by dereuromark** [comment]
:+1:

File: src/Shell/MigrationsShell.php [hunk 1]
```diff
@@ -35,7 +35,12 @@ public function getOptionParser()
             ->addOption('plugin', ['short' => 'p'])
             ->addOption('target', ['short' => 't'])
             ->addOption('connection', ['short' => 'c'])
-            ->addOption('source', ['short' => 's']);
+            ->addOption('source', ['short' => 's'])
+            ->addOption('ansi')
+            ->addOption('no-ansi')
+            ->addOption('version')
+            ->addOption('version', ['short' => 'V'])
```
Comment (dereuromark) [inline src/Shell/MigrationsShell.php]: Does it need this option twice here?

Comment (lorenzo) [inline src/Shell/MigrationsShell.php]: No, I was derp

**Pull request merged**